docs/src/Icon.doc.js

```diff
@@ -51,6 +51,10 @@ card(
         type: 'boolean',
         defaultValue: false,
       },
+      {
+        name: 'dangerouslySetSvgPath',
+        type: `{ __path: string }`,
```

Comment:
@akleiner2 could we add a description that the viewbox around the path should be 24x24?